docs/kyma/docs/017-details-local-installation.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Local installation
+type: Details
+---
+
+Simple as it may seem, the Kyma installation is a complex and dynamic process. This document extends the “031-gs-local-installation.md” guide with a detailed breakdown of the alternative installation method, i.e. the “run.sh” script.
+
+> **NOTE:** the ‘run.sh’ script should be used exclusively for development purposes.
+
+To start the local installation, run the following command:
+```
+./installation/cmd/run.sh
+```
+
+This script sets up default parameters, starts Minikube, builds Kyma-Installer, generates local configuration, creates the `installation` Custom Resource and sets up Installer. Subsequent sections provide a detailed description of each step.
+
+You can execute the `installation/cmd/run.sh` script with the following parameters:
+
+- `--skip-minikube-start` which skips execution of the `installation/scripts/minikube.sh` script. See the **Start Minikube** section for more details
+- `--vm-driver` points to either `virtualbox` or `hiperkit` depending on your operating system
```

Comment:
should be `which points` (my fault, missed that consistency during the first review) + period at the end?

Reply:
done